diff --git a/cmake/llvm-package.cmake b/cmake/llvm-package.cmake
index 958b914..3026bf3 100644
--- a/cmake/llvm-package.cmake
+++ b/cmake/llvm-package.cmake
@@ -1,6 +1,6 @@
 
-set(AnyDSL_PKG_LLVM_VERSION "10.0.1" CACHE STRING "LLVM version of AnyDSL")
-set(AnyDSL_PKG_RV_TAG "origin/release/10.x" CACHE STRING "LLVM is build with this git tag of RV")
+set(AnyDSL_PKG_LLVM_VERSION "12.0.0" CACHE STRING "LLVM version of AnyDSL")
+set(AnyDSL_PKG_RV_TAG "origin/release/12.x" CACHE STRING "LLVM is build with this git tag of RV")
 
 set(AnyDSL_PKG_LLVM_URL "https://github.com/llvm/llvm-project/releases/download/llvmorg-${AnyDSL_PKG_LLVM_VERSION}/llvm-project-${AnyDSL_PKG_LLVM_VERSION}.tar.xz" CACHE STRING "where to download LLVM")
 set(AnyDSL_PKG_RV_URL "https://github.com/cdl-saarland/rv" CACHE STRING "where to clone RV")
